app/src/main/java/org/mozilla/fenix/wallpapers/Wallpaper.kt

```diff
@@ -10,10 +10,20 @@ package org.mozilla.fenix.wallpapers
  * @property portraitPath A file path for the portrait version of this wallpaper.
  * @property landscapePath A file path for the landscape version of this wallpaper.
  * @property isDark Indicates if the most predominant color on the wallpaper is dark.
+ * @property themeCollection The theme collection this wallpaper belongs to.
  */
 data class Wallpaper(
     val name: String,
     val portraitPath: String,
     val landscapePath: String,
-    val isDark: Boolean
+    val isDark: Boolean,
+    val themeCollection: WallpaperThemeCollection,
```

Comment:
@Amejia481 What do you think of using a sealed class like this to track the theme type? Seems pretty extensible for the future to me

Reply:
Looks good 👍🏽